src/rollup-plugin-obfuscator.ts

```diff
@@ -50,24 +50,15 @@ function rollupPluginObfuscator (override?: Partial<RollupPluginObfuscatorOption
 	return {
 		name: 'rollup-plugin-obfuscator',
 
-		transform: options.global ? undefined : (code, id) => {
-			if (!filter(id)) return null;
+		renderChunk: (code, chunk) => {
+			// Make sure only to obfuscate JS files in the final output
+			if (!filter(chunk.fileName)) return null;
```

Comment:
I'm pretty sure chunks are always JS. So this filter isn't useful (and may cause issues if the user changes the `include`/`exclude` parameters).